src/rcamera.h

```diff
@@ -107,17 +115,19 @@
 extern "C" {            // Prevents name mangling of functions
 #endif
 
-#if defined(CAMERA_STANDALONE)
-void SetCameraMode(Camera camera, int mode);                // Set camera mode (multiple camera modes available)
-void UpdateCamera(Camera *camera);                          // Update camera position for selected mode
-
-void SetCameraPanControl(int keyPan);                       // Set camera pan key to combine with mouse movement (free camera)
-void SetCameraAltControl(int keyAlt);                       // Set camera alt key to combine with mouse movement (free camera)
-void SetCameraSmoothZoomControl(int szoomKey);              // Set camera smooth zoom key to combine with mouse (free camera)
-void SetCameraMoveControls(int keyFront, int keyBack,
-                           int keyRight, int keyLeft,
-                           int keyUp, int keyDown);         // Set camera move controls (1st person and 3rd person cameras)
-#endif
+Vector3 GetCameraForward(Camera3D* camera);
+Vector3 GetCameraUp(Camera3D* camera);
+Vector3 GetCameraRight(Camera3D* camera);
+void CameraMoveForward(Camera3D* camera, float distance, bool moveInWorldPlane);
+void CameraMoveUp(Camera3D* camera, float distance);
+void CameraMoveRight(Camera3D* camera, float distance, bool moveInWorldPlane);
+void CameraZoom(Camera3D* camera, float delta);
+void CameraYaw(Camera3D* camera, float angle, bool rotateAroundTarget);
+void CameraPitch(Camera3D* camera, float angle, bool lockView, bool rotateAroundTarget, bool rotateUp);
+void CameraRoll(Camera3D* camera, float angle);
+void CameraViewBobbing(Camera3D* camera);
```

Comment:
If the user has to call this on it's own to get bobbing and not part of update, I don't think the bob state should be part of the camera itself. Perhaps a separate structure for camera state that needs to be passed in.
Really the '0' to disable on the sway count is not correct, what disables bobbing is not calling this function.
Because of this I would not couple the bob state to the camera, since it's only used by this function. It's extra data that other camera systems don't need. Bobbing is such a minor feature I think that people who want it should have to track it's state separately.